specification/attestation.adoc

```diff
@@ -65,7 +69,7 @@ The TCB elements for each of them is summarized in the following table:
 .4+.^|Platform
     <| HW RoT for boot, measurement and storage
  .4+<| All M-mode firmwares, including the TSM-driver
-    <| All CPU hardware logic, including MMU and caches
+    <| All CPU hardware logic, including Debug, MMU and caches
```

Comment:
Maybe we just say All CPU hardware logic  - instead of stating individual sub-systems

Reply:
done